### PR TITLE
Llr callbacks testing

### PR DESCRIPTION
**Pull request by cyberbono3**
Tests for `gossip/c_llr_callbacks.go`:

- ProcessFullBlockRecord
- ProcessFullEpochRecord
- ProcessEpochVote
- ProcessBlockVotes

### CODE REVIEW

File: gossip/c_llr_process_votes_test.go [hunk 1]
```diff
@@ -129,6 +158,30 @@ func (s *LLRCallbacksTestSuite) Test_processBlockVoteLLRVotingDoubleSignIsMet()
...
+	e, err := em.EmitEvent()
+	s.Require().NoError(err)
+	bv2 := inter.AsSignedBlockVotes(mutableEventPayloadFromImmutable(e))
+
+	bv2.Val.Start = bs.LastBlock.Idx
+	bv2.Val.Epoch = 1
+	bv2.Val.Votes = []hash.Hash{hash.HexToHash("0x02"), hash.HexToHash("0x03")}
+
+
+	s.Require().NoError(s.env.ProcessBlockVotes(bv1))
```
Comment (cyberbono3) [inline gossip/c_llr_process_votes_test.go:182]: Any ideas how can I return errLLRVotingDoubleSign on L182 or L183?

Comment (cyberbono3) [reply gossip/c_llr_process_votes_test.go]: @uprendis @rus-alex I would appreciate some guidance regarding calling this test by set of validators.

File: gossip/c_llr_callbacks_test.go [hunk 2]
```diff
@@ -0,0 +1,1098 @@
...
+		},
+		{"block range 1-1000",
+			func() {
+				crit = defaultCrit
+			},
+			true,
+		},
+		{"block range 1-1000 and first topic",
+			func() {
+				require.NoError(r.t, err)
```
Comment (hadv) [inline gossip/c_llr_callbacks_test.go:615]: it seems that this require is not intended here, right?

```suggestion
```

**Comment by cyberbono3** [comment]
@rus-alex @uprendis can this be merged?

**Pull request merged**